src/SFA.DAS.PR.Application.UnitTests/Requests/Commands/CreateNewAccountRequest/CreateNewAccountRequestCommandValidatorTests.cs

```diff
@@ -81,20 +84,92 @@ public async Task CreateNewAccountRequestCommand_Invalid_EmployerContactEmail()
         );
 
         var result = await sut.TestValidateAsync(new CreateNewAccountRequestCommand
-            {
-                Ukprn = 10000003,
-                EmployerPAYE = "EmployerPAYE",
-                RequestedBy = Guid.NewGuid().ToString(),
-                EmployerContactEmail = "invalid_email",
-                Operations = [Operation.CreateCohort],
-                EmployerOrganisationName = "EmployerOrganisationName",
-                EmployerContactFirstName = "EmployerContactFirstName",
-                EmployerContactLastName = "EmployerContactLastName",
-                EmployerAORN = "EmployerAORN"
-            }
+        {
+            Ukprn = 10000003,
+            EmployerPAYE = Paye,
+            RequestedBy = Guid.NewGuid().ToString(),
+            EmployerContactEmail = "invalid_email",
+            Operations = [Operation.CreateCohort],
+            EmployerOrganisationName = "EmployerOrganisationName",
+            EmployerContactFirstName = "EmployerContactFirstName",
+            EmployerContactLastName = "EmployerContactLastName",
+            EmployerAORN = "EmployerAORN"
+        }
         );
 
         result.ShouldHaveValidationErrorFor(q => q.EmployerContactEmail)
             .WithErrorMessage(CreateAddAccountRequestCommandValidator.EmployerContactEmailValidationMessage);
     }
+
+    [TestCase("")]
+    [TestCase(null)]
+    public async Task CreateAddAccountRequestCommand_No_Paye(string? paye)
+    {
+        var sut = new CreateNewAccountRequestCommandValidator(
+            _requestReadRepositoryValidMock.Object,
+            _providersReadRepositoryValidMock.Object
+        );
+        var result = await sut.TestValidateAsync(new CreateNewAccountRequestCommand
+        {
+            Ukprn = 10000003,
+            EmployerPAYE = paye,
+            RequestedBy = Guid.NewGuid().ToString(),
+            EmployerContactEmail = "test@test.com",
+            Operations = [Operation.CreateCohort],
+            EmployerOrganisationName = "EmployerOrganisationName",
+            EmployerContactFirstName = "EmployerContactFirstName",
+            EmployerContactLastName = "EmployerContactLastName",
+            EmployerAORN = "EmployerAORN"
+        }
+        );
+
+        result.ShouldHaveValidationErrorFor(q => q.EmployerPAYE)
+            .WithErrorMessage(CreateNewAccountRequestCommandValidator.NoPayeMessage);
+    }
+
+    [TestCase("1/1")]
+    [TestCase("11/1")]
+    [TestCase("11A/1")]
+    [TestCase("A11/1")]
+    [TestCase("1AA/1")]
+    [TestCase("11A/1")]
+    [TestCase("A1A1")]
+    [TestCase("1")]
+    [TestCase("12")]
+    [TestCase("123/")]
+    [TestCase("11A/1234567")]
+    [TestCase("1A1/1234567")]
+    [TestCase("A11/12345678")]
+    [TestCase("AAA")]
+    [TestCase("AAA/")]
+    [TestCase("AAA/1")]
+    [TestCase("AAA/12")]
+    [TestCase("A1A/1234567")]
+    [TestCase("A11/12345678")]
+    [TestCase("A11/123456789")]
+    [TestCase("A11/1234567890")]
```

Comment:
Instead of repeating this test twice, you could have added these test on the Regex itself so we do not need to then retest it in each validator that uses the regex

Reply:
I've isolated the test cases into a fixture and the two sets of test are using a single fixture, so no duplication.